### ISSUE TITLE
CImg directory blank after git clone

### ISSUE THREAD

**Issue by kennbr34** [eddieantonio/imgcat#37]
I noticed when I cloned the repository using the 'git clone' the directory 'CImg' was empty, which caused problems with compilation when 'CImg.h' was needed.

**Comment by eddieantonio** [comment]
A recent PR #34 included Cimg.h via submodules. To clone it properly, one needs to use this command:

```sh
git clone --recurse-submodules https://github.com/eddieantonio/imgcat.git
```

I should add this to the `README`...

**Comment by eddieantonio** [comment]
Thanks for the issue! I updated the `README`

Fixed by  580543f53d95b11478b9c85220f7a77bde0a7a37
